### ISSUE TITLE
Html loader overrides template variables

### ISSUE THREAD

**Issue by kof** [styleguidist/react-styleguidist#312]
I have an html loader in my webpack.config:

```js
{
 test: /\.html$/,
 loader: 'html-loader'
}
```

Now document template renders this:

```html
<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<title><%=htmlWebpackPlugin.options.title%></title>
	</head>
	<body>
		<div id="app"></div>
	<script type="text/javascript" src="build/bundle.js"></script></body>
</html>
```

As we discussed with @sapegin the issue seems to be [here](https://github.com/jantimon/html-webpack-plugin/issues/540) but a solution needs to be more general.

**Comment by sapegin** [comment]
[More info in FAQ](https://github.com/jantimon/html-webpack-plugin/blob/master/docs/template-option.md)

Probably we can hardcode EJS loader from html-webpack-plugin, something like this:

```js
const loader = require.resolve('html-webpack-plugin/lib/loader');
new HtmlWebpackPlugin({
  template: `!!${loader}!${template}`
})
```

Another solution would be to replace html-webpack-plugin with something simpler that can’t be affected by user loaders configuration.

/cc @bebraw

**Comment by kof** [comment]
What about other things, can anything else in the users wp config break sg?

**Comment by sapegin** [comment]
Only practice can tell ;-)

**Comment by kof** [comment]
Well anything you rely on from the webpack config may be different in the users one so, its basically guaranteed that something can go wrong, I am not familiar with the code base yet to see what sg depends on.

**Comment by sapegin** [comment]
That’s why I didn’t want to do any user config importing ;-) But we had the same issues before, even worse.

**Comment by sapegin** [comment]
Should be fixed in react-styleguidist@5.0.0-beta.11.
